README.md

```diff
@@ -109,7 +134,9 @@ Example   | Description |
 
 ### JavaScript
 
-[(Click here to go back to the main list)](#outline)
+[(Click here to go back to the library)](#example-library)
+
+[Try Pulumi Copilot](https://app.pulumi.com/?utm_campaign=pulumi-examples-github-repo&utm_source=github.com&utm_medium=pulumi-examples) and use natural-language prompts to generate Pulumi example programs in _any_ language.
```

Comment:
This generally feels repetitive and advertising-y/spammy to me. 

I'd also suggest putting these at the end of each list with something like "Don't see what you're looking for? [Try Pulumi AI]" --- and then IMO leaving off the "and use ..." bit to keep it simple, as users will almost surely be able to gather from context what to expect when they get there.